### ISSUE TITLE
Breadcrumb separator has insufficient contrast

### ISSUE THREAD

**Issue by adampage** [adampage/github-cli-test#7]
The Breadcrumb's “/” separator is visually informative and therefore must have at least 4.5:1 contrast to conform with [WCAG 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum).

## Applicable WCAG Success Criteria

- [`1.4.3` Contrast (Minimum) — Level AA](https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum)
